wayland-client/build.rs

```diff
@@ -10,5 +10,6 @@ fn main() {
     let out_dir_str = var("OUT_DIR").unwrap();
     let out_dir = Path::new(&out_dir_str);
 
+    println!("cargo:rerun-if-changed={}", protocol_file);
```

Comment:
You need to call `.display()` on the paths to get an object that can actually be fed to `println!()`.

Reply:
This is not the part of the code where `protocol_file` is a `&Path`, but you're right of course. I apparently didn't test locally :man_facepalming: